public/locales/fr/translation.json

```diff
@@ -104,6 +104,7 @@
         "swapMessage": "Acceptez sur votre wallet pour continuer",
         "wrapMessage": "Pas de frais de protocole pour les swaps ETH/WETH",
         "submitted": "Transaction soumise",
+        "transactionCompleted": "Transaction terminée",
```

Comment:
These translation.json files will be overwritten by our translation too. You only have to do the english one for now.

Reply:
Okay sounds good I was unsure whether or not I had to change the other translations